src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -279,6 +280,35 @@ private static int getSchemaDelay()
 
     public static final StorageService instance = new StorageService();
 
+    private final java.util.function.Predicate<Keyspace> anyOutOfRangeOpsRecorded
+            = keyspace -> keyspace.metric.outOfRangeTokenReads.getCount() > 0
+                          || keyspace.metric.outOfRangeTokenWrites.getCount() > 0
+                          || keyspace.metric.outOfRangeTokenPaxosRequests.getCount() > 0;
```

Comment:
TODO: rearrange `||`s

Reply:
wdym ?

Reply:
Ignore. This comment should have gone in the 5.0 PR.